BazelExtensions/extensions.bzl

```diff
@@ -324,10 +324,13 @@ def _make_headermap_impl(ctx):
 
     # Extract propagated headermaps
     for hdr_provider in ctx.attr.deps:
-        if not hasattr(hdr_provider, "objc"):
-            continue
+        hdrs = []
 
-        hdrs = hdr_provider.objc.direct_headers
+        if CcInfo in hdr_provider:
+            hdrs.extend(hdr_provider[CcInfo].compilation_context.headers.to_list())
```

Comment:
Some targets do return the CcInfo provider with an embedded CompilationContext that should have the same headers that were previously attached to ObjcProvider -- but doing this alone fails with error
<img width="1680" alt="Screen Shot 2021-03-11 at 10 47 07 AM" src="https://user-images.githubusercontent.com/1711204/110853843-f3988200-8268-11eb-9f84-658b404a4994.png">

Reply:
This makes sense, I think what you've done here and seems more aligned with what we were doing before as opposed to only passing `direct_headers`.  It may have been failing under `darwin-sandbox` if it didn't get all the headers.